Makefile

```diff
@@ -12,8 +12,8 @@ install:  ## Install the stack
 
 ##@ Tests
 
-
-test: typecheck test-integration  ## Run all the tests
+test: typecheck test-integration test-doc ## Run all the tests
+	poetry run pytest --doctest-modules avatars
```

Comment:
this line test the docstring with pytest, it also run all test in `avatar`

Reply:
that's cool!

Reply:
Are you sure this `test-doc` is doing something? I see it in no Makefile